### PR TITLE
Added Toast component

### PR DESCRIPTION
**Pull request by PRABgit01**
### Simple Action Toast 
Screenshot 2023-12-24 at 1 38 06 PM

### CODE REVIEW

File: src/stories/Toast/Toast.stories.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export default {
+  title: "Toast ",
+};
+
+export const SimpleActionToast = () => (
+  <ToastContainer>
+    <Toast text="Successful toast for user action." type="success" />
+    <Toast text="Warning toast for user action." type="warning" />
+    <Toast text="danger toast for user action." type="dangerHover" />
```
Comment (tarakhant1297) [inline src/stories/Toast/Toast.stories.js:13]: @PRABgit01 give proper name to `dangerHover `

File: src/stories/Toast/Toast.style.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import theme from "../theme";
+
+export const ToastStyle = styled.button`
```
Comment (tarakhant1297) [inline src/stories/Toast/Toast.style.js:4]: @PRABgit01 add Styled add in prefix instead of suffix

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Toast/Toast.stories.js (hunk 1 above)
Comment (PRABgit01) [reply src/stories/Toast/Toast.stories.js:13]: ok

File: src/stories/Toast/Toast.style.js (hunk 2 above)
Comment (PRABgit01) [reply src/stories/Toast/Toast.style.js:4]: ok

**Review by tarakhant1297 — APPROVED** [review]

**Pull request merged**